happi/cli.py

```diff
@@ -297,45 +295,45 @@ def edit(ctx, name: str, edits: List[str]):
     client = ctx.obj
 
     logger.debug('Starting edit block')
-    item = client.find_item(name=name)
+    item = None
+    try:
+        item = client.find_item(name=name)
+    except SearchError as e:
+        raise click.ClickException(str(e))
+
     if len(edits) < 1:
-        click.echo('No edits provided, aborting')
-        raise click.Abort()
+        raise click.ClickException('No edits provided')
 
     for edit in edits:
         field, value = edit.split('=', 1)
         # validate field
         try:
             field = is_valid_identifier_not_keyword(field)
         except ValueError:
-            logger.error(f'field ({field}) is not a valid keyword')
-            raise click.Abort()
+            raise click.ClickException(f'field ({field}) is not a valid keyword')
```

Comment:
The message was a bit off - this checks to see that it's _not_ a Python keyword. And the parentheses around it are a bit awkward. How about:
```suggestion
            raise click.ClickException(f'field {field!r} is not a valid identifier')
```